docs/tutorial-module/run-n-test.rst

```diff
@@ -131,20 +131,20 @@ Now, go into the root directory of your module, and run
 
 .. code-block:: bash
 
-    (my-module-venv)$ git remote add origin URL-OF-YOUR-GITHUB-REPO
+    git remote add origin URL-OF-YOUR-GITHUB-REPO
```

Comment:
All the commands of this file are missing a "$" before to be consistent with the rest of the documentation.